### ISSUE TITLE
CashFusion Wallet Settings typo

### ISSUE THREAD

**Issue by JLMoriart** [Electron-Cash/Electron-Cash#1924]
Under "Self-Fusing" it currently reads "Allow this wallet to participate **multiply** in the same fusion round?"

Should probably read something along the lines of "Allow this wallet to act as two separate participants in the same fusion round?"

It would also then be useful to have a little bit of explanatory text beneath. Something like "Enabling this option reduces your privacy, but makes it more likely to get Fusions if not many others are currently running CashFusion".

**Comment by cculianu** [comment]
You're right about the explanatory text -- we can add that.  Perhaps as a tool-tip or as a little `?` button you click.  

I'm hesitant to add to the wording in the UI itself to the longer version you suggest.  It will just lead to a lot of text in the UI, which is a bit of a no-no.  It's always a challenge not to put too much text in a UI.  UIs with very wordy paragraphs of text tend to look ... not as good as those with less text.

**Comment by JLMoriart** [comment]
> You're right about the explanatory text -- we can add that. Perhaps as a tool-tip or as a little `?` button you click.
> 
> I'm hesitant to add to the wording in the UI itself to the longer version you suggest. It will just lead to a lot of text in the UI, which is a bit of a no-no. It's always a challenge not to put too much text in a UI. UIs with very wordy paragraphs of text tend to look ... not as good as those with less text.

A little question mark tooltip sounds reasonable to me for the extra info!

**Comment by zander** [comment]
Suggestion:

1. rename groupbox title from "self-fusing" to "Fuse Faster"
2. Change label to: "Faster fusing gets multiple fuses from this wallet in one transaction, possibly lowering privacy".
3. Change the combobox labels to: ["Faster", "Saver"]. Where 'saver' is still the default.
4. Add a question mark button with the point that this is relevant only when there are not enough participants.

This makes the choice about something that the end user actually cares about. Not about the technical details they don't care about.

**Comment by cculianu** [comment]
You’re probably right. @fyookball — what do you think?
